src/main/java/frc/robot/Robot.java

```diff
@@ -102,12 +102,19 @@ public void robotInit() {
         SmartDashboard.putData("Autonomous Test", drive.autonomousCommand("Slolom"));
 
         autoChooser.setDefaultOption("Test", drive.autonomousCommand("Test"));
+        autoChooser.setDefaultOption("PathPlan Test", drive.autonomousCommand("Test", true, false));
         autoChooser.addOption("Slolom", drive.autonomousCommand("Slolom"));
+        autoChooser.addOption("PathPlan Slolom", drive.autonomousCommand("Slolom", true, false));
         autoChooser.addOption("Barrel", drive.autonomousCommand("Barrel"));
+        autoChooser.addOption("PathPlan Barrel", drive.autonomousCommand("Barrel", true, false));
         autoChooser.addOption("CollectBlue", drive.autonomousCommand("CollectBlue"));
         autoChooser.addOption("CollectRed", drive.autonomousCommand("CollectRed"));
         autoChooser.addOption("Bounce", drive.autonomousCommand("Bounce1").andThen(drive.autonomousCommand("Bounce2"))
                 .andThen(drive.autonomousCommand("Bounce3")).andThen(drive.autonomousCommand("Bounce4")));
+        autoChooser.addOption("PathPlan Bounce",
+                drive.autonomousCommand("Bounce1", true, false).andThen(drive.autonomousCommand("Bounce2", true, false))
```

Comment:
One thing I noticed is that `x.andThen(y).andThen(z).andThen(q)` can just be done as `x.andThen(y, z, q)` and will produce a more "flat" command structure.